protos/eventpb/eventpb.proto

```diff
@@ -59,6 +59,8 @@ message Event {
   // In this case, the WAL append event would be this event
   // and the next field would contain the message sending event.
   repeated Event next = 100;
+
+  string dest_module = 200;
```

Comment:
I think it could be set for all events by the Node itself. This way all events would have both the recipient and the sender. Whether to use this information not would be up to the event consumer. But it's just a suggestion to unify code, it's up to you.

Reply:
This would (slightly) complicate the interface of the `DslModule` because all handlers would have to accept the source as a separate parameter.

A bit of context: `DslModule` is a construction that I want to propose to add, which will allow writing sequential passive modules in a more declarative pseudocode-like way. I will make a PR with it shortly after this PR is merged.